dotnet/samples/Extism.Sdk.Sample/Program.cs

```diff
@@ -15,15 +15,5 @@
 Console.Write("Input: ");
 var input = Console.ReadLine() ?? "";
 
-var response = plugin.CallFunction("count_vowels", Encoding.UTF8.GetBytes(input));
-if (response == 0)
-{
-    var output = Encoding.UTF8.GetString(plugin.OutputData());
-    Console.WriteLine(output);
-}
-else
-{
-    Console.WriteLine($"Status code: {response}");
-    Console.WriteLine($"Context error: {context.GetError()}");
-    Console.WriteLine($"Plugin error: {plugin.GetError()}");
-}
\ No newline at end of file
+var output = plugin.CallFunction("count_vowels", Encoding.UTF8.GetBytes(input));
+Console.WriteLine(Encoding.UTF8.GetString((output)));
```

Comment:
Unfortunately we can't pin [Span<T>](https://learn.microsoft.com/en-us/archive/msdn-magazine/2018/january/csharp-all-about-span-exploring-a-new-net-mainstay) (because it's just a view on a contagious region of memory and the implicity generated Main method here will be async because we have used an async method (`await File.ReadAllBytesAsync`).

We can directly convert the bytes to a string and then assign it to output instead:
```suggestion
var output = Encoding.UTF8.GetString(plugin.CallFunction("count_vowels", Encoding.UTF8.GetBytes(input)));
Console.WriteLine(output);
```
Or can also use `File.ReadAllBytes` and the method will not be async anymore.